libs/npm/core/src/hooks/api-editor/editor/useApiContents.ts

```diff
@@ -134,9 +134,16 @@ export const useApiContents = (
     },
     stream: async (path: string) => {
       const params = { path };
-      return await api.get<any, AxiosResponse<any>, any>(
+      const response = await api.get<any, AxiosResponse<any>, any>(
```

Comment:
Can we update the `any` types to a `byte[]`?

Reply:
Unfortunately, we cannot do that in this case.